tests/acceptance/expected-failures-on-OWNCLOUD-storage.txt

```diff
@@ -1964,6 +1969,12 @@ apiShareUpdateToShares/updateShare.feature:155
 apiShareCreateSpecialToShares2/createShareDefaultFolderForReceivedShares.feature:21
 apiShareCreateSpecialToShares2/createShareDefaultFolderForReceivedShares.feature:22
 #
+# https://github.com/owncloud/product/issues/276
+#
+# Trying to decline a received share will give status 500 error.
+#
+webUISharingAcceptShares/acceptShares.feature:85
```

Comment:
This is the first webUI scenario to be added to this expected-failures file. This expected-failures file is (currently) only used for the API tests.

Is there a webUI test failing somewhere? That needs to be in an expected-failures file?

Reply:
Actually, this is an issue from `owncloud/product` # https://github.com/owncloud/product/issues/276
I will if it actually fails in CI..

Reply:
Test is running fine.. I've made a comment to the issue if it can be closed.
https://drone.owncloud.com/owncloud/ocis/2263/45/6